### DISCUSSION TITLE
Version 50 for Animate?

### DISCUSSION THREAD

**Discussion by henke37** [airsdk/Adobe-Runtime-Support#2150] [General]
Are there plans for letting Animate users use the new runtime? I'm interested in the new features.

**Comment by ajwfrost** [comment]
It should be possible to add the new SDK into Animate using the normal "Manage AIR SDKs" option from the Help menu.

But, it looks like we missed updating a sample XML file which Animate are using as the template for their application descriptor files (rather than them using the template XML files..) -> see #2147 and the workaround there..

thanks

**Comment by Jason0247** [comment]
@ajwfrost Hello, the problem that Animate cannot be used can only be used until the next AIR version, right?

**Reply by ajwfrost** [reply]
Hi -> there's a workaround for this that you can manually apply to the AIR SDK 50.0.0.1 (see link above) but otherwise yes, we'll have this sorted out in our next release coming shortly...

thanks
